### PR TITLE
Apply feedback

### PR DESCRIPTION
**Pull request by andrewdes**
Addressing feedback in #24

### CODE REVIEW

**Comment by yeekangc** [comment]
The front matter and the actual title plus description aren't consistent with each other?  @andrewdes @gkwan-ibm, please take another look.

**Comment by gkwan-ibm** [comment]
Should Helm instead of helm?

File: README.adoc [hunk 1]
```diff
@@ -39,33 +39,29 @@ Explore how to push Docker images to a custom registry and run them inside of co
...
+
+Kubernetes is an open source container orchestrator that automates many tasks involved in 
+deploying, managing, and scaling containerized applications. If you would like to learn
+more about Kubernetes, check out the https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^]
+guide.
+
+Various clouds offer solutions for running your workloads in a {kube} cluster. Using a 
+cloud-based infrastructure allows you to focus on developing your microservices rather 
+than worrying about details that are related to the servers you deploy them to. Using 
+cloud allows you to easily scale your microservices and serve them in a high-availability
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
a cloud allows you to easily scale your microservices and serve them in a high-availability 
```

File: README.adoc [hunk 2]
```diff
@@ -39,33 +39,29 @@ Explore how to push Docker images to a custom registry and run them inside of co
...
+cloud-based infrastructure allows you to focus on developing your microservices rather 
+than worrying about details that are related to the servers you deploy them to. Using 
+cloud allows you to easily scale your microservices and serve them in a high-availability 
+setup.
 
 IBM Cloud offers a hosted Kubernetes service that is called IBM Cloud Kubernetes Service (IKS). 
 IKS is a service that is offered as part of IBM's public cloud offerings. It provides a 
-hosted {kube} cluster that you can deploy your microservices to. You will use it 
+hosted {kube} cluster that you can deploy your microservices to. You will use IKS 
 with IBM Cloud Container Registry.
```
Comment (yeekangc) [inline README.adoc]: I will move "You will use IKS with IBM Cloud Container Registry" to below.  Best to have this in the last paragraph where we talk about what you will do.

Comment (yeekangc) [reply README.adoc]: Also, we should explain what is a container registry and why at an appropriate location in the guide.

File: README.adoc [hunk 3]
```diff
@@ -778,14 +780,14 @@ Some situations may require you to manage your own infrastructure, such as if yo
...
-https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is a {kube} platform. A private 
-cloud may be a more attractive option to those who must run their services on 
+A private cloud may be a more attractive option to those who must run their services on 
 infrastructure that is controlled by them. While the operations team will still need to 
 maintain this infrastructure, a private cloud still provides benefit to developers so 
 they don't need to think about infrastructure. {kube} acts as an abstraction of the 
 infrastructure so the developers can request what they need and it is the responsibility 
 of the infrastructure's maintainers to provide the necessary resources.
 
+https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is a {kube} platform.
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is a on-premises {kube} platform from IBM. 
```

File: README.adoc [hunk 4]
```diff
@@ -778,14 +781,14 @@ Some situations may require you to manage your own infrastructure, such as if yo
...
-https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is a {kube} platform. A private 
-cloud may be a more attractive option to those who must run their services on 
+A private cloud may be a more attractive option to those who must run their services on 
 infrastructure that is controlled by them. While the operations team will still need to 
 maintain this infrastructure, a private cloud still provides benefit to developers so 
 they don't need to think about infrastructure. {kube} acts as an abstraction of the 
 infrastructure so the developers can request what they need and it is the responsibility 
 of the infrastructure's maintainers to provide the necessary resources.
 
+https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is a on-premises {kube} platform from IBM.
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
https://www.ibm.com/cloud/private[IBM Cloud Private^] (ICP) is an on-premises {kube} platform from IBM. 
```

**Comment by yeekangc** [comment]
We need to tie things together better still for this paragraph but we can file a separate issue for it:
_You will use Helm to deploy these microservices to IBM Cloud using Open Liberty Helm charts.
Helm is a package manager for {kube}. It uses templates to generate {kube} yaml files and then
deploys and manages them as releases in your cluster. You will use IKS with IBM Cloud Container Registry.
IBM Cloud Container Registry is a private registry used to store and distribute your
container images._

**Comment by andrewdes** [comment]
I have filed an issue to track that here: https://github.com/OpenLiberty/draft-guide-cloud-ibm/issues/28

**Pull request merged**